modules/unsupported/wps/src/main/java/org/geotools/data/wps/WPSProcess.java

```
@@ -124,20 +126,28 @@ protected WPSProcess(ProcessFactory factory) {
 				exeRequest.addInput(identifier, list);
 			}
 		}
-		
+		LOGGER.info(exeRequest.toString());
 		// send the request and get the response
 		ExecuteProcessResponse response;
 		try {
 			response = wps.issueRequest(exeRequest);
 		} catch (ServiceException e) {
+			LOGGER.info(e.getMessage());
 			return null;
 		} catch (IOException e) {
+			LOGGER.info(e.getMessage());
 			return null;
 		}
 		
 		// if there is an exception in the response, return null
 		// TODO:  properly handle the exception?
-		if (response.getExceptionResponse() != null || response.getExecuteResponse() == null) {
+		if (response.getExceptionResponse() != null) {
```

Wouldn't it be better to throw an exception instead?